### DISCUSSION TITLE
YAML syntax error shows accompanying "TypeError: Error.captureStackTrace is not a function" on page

### DISCUSSION THREAD

**Discussion by eric-anderton-at-rearc** [noop-inc/documentation#137] [Bugs] [answered]
I see this error message when typing in a YAML syntax error into a YAML field.  Seen on Pipeline Condition and Workflow fields.

<img width="257" alt="firefox_C52Shx9RDD" src="https://github.com/noop-inc/documentation/assets/136380266/8fd59bef-ba1b-4c8a-b19e-c285a9f6fdbd">

To recreate:

1. Open Pipeline Settings
1. Navigate to Condition
1. Enter non-YAML nonsense into field on its own line (note angry-looking red (X) in the margin)
1. See "TypeError: Error.captureStackTrace is not a function" just below the input textarea

Notes:

IMO, clearly linked to whatever YAML parser is in use.  Removing the offending bad YAML also clears the error message.

**Comment by derekwolpert** [comment]
@eric-anderton-at-rearc Do you have any context on what input value was causing this issue? I'm unable to reproduce this TypeError encounter on my end.

Below is a screenshot showing how a YAML error should be presented.

<img width="369" alt="Screenshot 2024-03-18 at 10 35 25 PM" src="https://github.com/noop-inc/documentation/assets/48101033/675e3080-1efa-432d-b8be-d2f62ba09d6c">

**Reply by eric-anderton-at-rearc** [reply]
@derekwolpert - thanks for looking into this.  It's possible that this is a browser/OS version issue. I'm running Firefox 123.0.1 on Windows 11.

![image](https://github.com/noop-inc/documentation/assets/136380266/1ba25927-9ebb-4cf4-b01c-fff1b6f45010)

![image](https://github.com/noop-inc/documentation/assets/136380266/b0848e74-f80f-4c04-83e1-c1e2e4d9a317)

**Comment by derekwolpert** [comment, marked as answer]
@eric-anderton-at-rearc Thanks for the additional context here. A fix has been deployed to https://noop.dev/, which should ensure error details are presented as expected in Firefox.

Please reach back out if you're still experiencing this issue, and we'll investigate further.

<img width="1004" alt="Screenshot 2024-03-19 at 12 59 07 PM" src="https://github.com/noop-inc/documentation/assets/48101033/6f3b8b97-a940-4bc6-852d-51d404cfbd86">
